chromium_src/chrome/browser/component_updater/component_updater_utils.cc

```diff
@@ -5,18 +5,13 @@
 
 #include "src/chrome/browser/component_updater/component_updater_utils.cc"
 
-#include "base/functional/callback.h"
-#include "chrome/browser/browser_process.h"
-#include "components/component_updater/component_updater_service.h"
+#include "brave/components/brave_component_updater/browser/brave_on_demand_updater.h"
 
 namespace component_updater {
 
 void BraveOnDemandUpdate(const std::string& component_id) {
```

Comment:
this function will be removed in a follow up PR. It's left for now to not overcomplicate the PR.